package-lock.json

```diff
@@ -1,9851 +1,8 @@
 {
   "name": "project-sayyara",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Are you using node 16? Why are you downgrading the `package-lock.json`?

Reply:
Fixed